legacy/marconi-chain-index-legacy/cardano-api-extended-legacy/src/Cardano/Api/Extended.hs

```diff
@@ -0,0 +1,24 @@
+{-# OPTIONS_GHC -Wno-missing-import-lists #-}
```

Comment:
Doesn't change anything I believe, but couldn't you just refer to the `cardano-api-extended` of `marconi-chain-index`

Reply:
I thought of that, but I was thinking it was possible we would want to add or modify `cardano-api-extended` in `marconi-chain-index` with breaking changes, which could require us to then update the `marconi-chain-index-legacy` code. It seemed better to avoid that development burden if it was not necessary.

But we can make a follow-up ticket to drop the `cardano-api-extended-legacy` and just refer to the original, if that's easier.